code/modules/reagents/oldchem/reagents/reagents_toxin.dm

```diff
@@ -26,6 +26,234 @@
 		return
 
 
+// Terror Spider, Green, Harmless Pheremone
+
+
+/datum/reagent/terror_green_toxin
```

Comment:
They're not limited to the awaymission - they work with terror spiders in general.
I've tried to keep the away mission code file to only things that are related to the away mission.